sw/device/tests/BUILD

```diff
@@ -1085,6 +1098,8 @@ opentitan_test(
     ],
 )
 
+# This is sim_dv only target.
+# Real sival target is implement after this.
```

Comment:
If it's simdv only, you should set `exec_env = { "//hw/top_earlgrey:sim_dv": None }` to save some CI build time.